.github/mergify.yml

```diff
@@ -0,0 +1,70 @@
+queue_rules:
+  - name: urgent
+    speculative_checks: 2
+    batch_size: 2
+    conditions:
+      - check-success=CI
+      - check-success=Coverage
```

Comment:
Test Coverage: Can we also check the Google Cloud test result for each queue and rule?
(I think it's called `pull-request`.)